NEWS.md

```diff
@@ -1,25 +1,59 @@
-### v9.7.0 (2022-11-14)
-
-* Added new configuration option, `grpc.ignore_status_codes`, which can be used to select nonzero gRPC status codes to ignore and not report as errors.
-
-### v9.6.0 (2022-11-09)
-
-* Dropped support for `vision`, and instead only instrument `@hapi/vision`.
-
+### v9.7.1 (2022-12-05)
+
+* Reintroduced throttled reading of dependency tree at startup to prevent EMFILE issues.
+
+* Improved Restify support
+  * Added a new test stanza to run restify >=10 on Node 18.
+  * Update our versioned tests to support Restify 9.0.0.
+
+* Enhanced agent to lay foundation for supporting for Code Level Metrics via Codestream. Note that while changes have been made to the agent for [Codestream](https://docs.newrelic.com/docs/codestream/how-use-codestream/performance-monitoring/), this integration is not fully finished and should not be used.
+  * Attached code level metrics attributes to middleware spans when `config.code_level_metrics.enabled` is true.
+  * Added configuration option to toggle Code Level Metrics: `code_level_metrics.enabled`.  
+    * It defaults to false.
+    * The environment variable is `NEW_RELIC_CODE_LEVEL_METRICS_ENABLED`.
+  * Added a utility to retrieve Code Level Metrics(CLM) metadata from a function reference.
+
+* Improved the readability and maintainability of agent by reducing the [Cognitive Complexity](https://www.sonarsource.com/resources/cognitive-complexity/) of various aspects of the agent.

```

Comment:
I think the entire nested bullet list here could be removed. Probably no one reading release notes cares what our source code looks like.